### PR TITLE
[tests-only] Change ocis storage driver from owncloud to ocis

### PR DESCRIPTION
**Pull request by saw-jan**
changes:
- change storage driver to `ocis`
- revert changes made in https://github.com/owncloud/owncloud-sdk/pull/755
- bump ocis commit-id

Closes https://github.com/owncloud/owncloud-sdk/issues/834

### CODE REVIEW

**Comment by individual-it** [comment]
redis also can be deleted

File: tests/providerTest.js [hunk 1]
```diff
@@ -223,9 +204,13 @@ describe('provider testing', () => {
...
-        chai.assert.strictEqual(
-          result.status, 200, `creating user '${parameters.username}' failed`
-        )
+
+        const { message } = getOCSMeta(result)
+        if (result.status !== 200 && message !== 'account already exists') {
+          chai.assert.strictEqual(
+            result.status, 200, `creating user '${parameters.username}' failed`
+          )
+        }
```
Comment (saw-jan) [inline tests/providerTest.js:213]: recreating a user should work but sometimes it fails with `account already exists` mostly with share user `testABC`.
So have to add this code block.

But will also work with this delete code if it is not to removed:
https://github.com/owncloud/owncloud-sdk/blob/8623274eed976753b03a6d31572d44b29b18b5a4/tests/providerTest.js#L213-L215

Comment (saw-jan) [reply tests/providerTest.js:213]: Related issue in ocis: https://github.com/owncloud/ocis/issues/2027

Comment (individual-it) [reply tests/providerTest.js:215]: the user should have done at least a request to `/ocs/v2.php/cloud/capabilities` , see the hack below. Maybe that is not enough.
Can we change the hack below an make it a webdav `PROPFIND` request? will that create the user correctly or do we need to create a file / folder?

Comment (saw-jan) [reply tests/providerTest.js:215]: @individual-it We need to make any requests twice and then only the user will be deleted successfully. (requesting `capabilities` twice will work)
Should we do that?

Comment (individual-it) [reply tests/providerTest.js:215]: yes, let's try that and please mention that also in owncloud/ocis#2027

**Review by individual-it — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

File: tests/providerTest.js (hunk 1 above)
Comment (phil-davis) [reply tests/providerTest.js]: Comment added to the issue: https://github.com/owncloud/ocis/issues/2027#issuecomment-834116526

**Pull request merged**